targets/gnome

```diff
@@ -8,9 +8,15 @@ HOSTBIN='startgnome'
 CHROOTBIN='crouton-noroot startgnome gnome-session-wrapper'
 . "${TARGETSDIR:="$PWD"}/common"
 
+if release -ge wheezy -ge kali -ge trusty; then
```

Comment:
I just did a quick search and it seems wheezy and kali is using -fallback so it should be `release -lt jessie -lt sana -lt trusty` or `release -ge jessie -ge sana -ge trusty`